perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -1270,9 +1270,13 @@ sub waste_munge_enquiry_data {
     $self->_set_user_source;
 }
 sub waste_get_next_dd_day {
-    my $self = shift;
+    my ($self, $payment_type) = @_;
+
+    # new DD mandates must have a 10-day wait
+    my $dd_delay = 10;
 
-    my $dd_delay = 10; # No days to set up a DD
+    # ad-hoc payments on an existing mandate only need a 5-day wait
```

Comment:
I'm sure this is okay, but where does this fact/change come from? I can't see anything in the PR text or elsewhere about it.